IbisLib/usacquisitionobject.cpp

```diff
@@ -983,29 +983,15 @@ void USAcquisitionObject::ExportTrackedVideoBuffer(QString destDir , bool masked
     //Prepare for writing out calibration matrix
     vtkMatrix4x4 *calMatrix = this->GetCalibrationTransform()->GetMatrix();
     QString calMatString;
-    calMatString.append( QString::number(calMatrix->GetElement( 0, 0 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 0, 1 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 0, 2 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 0, 3 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 1, 0 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 1, 1 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 1, 2 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 1, 2 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 2, 0 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 2, 1 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 2, 2 ), 'g', 6 ) );
-    calMatString.append( ", ");
-    calMatString.append( QString::number(calMatrix->GetElement( 2, 3 ), 'g', 6 ) );
+    for( int i = 0; i < 3; i++ )
+    {
+        for( int j = 0; j < 4; j++ )
+        {
+            calMatString.append( QString::number(calMatrix->GetElement( i, j ), 'g', 6 ) );
+            if( j < 3 )
+                calMatString.append( ", ");
```

Comment:
Not sure this works, the "," will not be in the right place.
I wonder if we shouldn't record the transform as in .xfm format, with a space separator and ends with ";". Something like:
Linear_Transform =
T_00 T_01 T02 T03
T_10 T_11 T12 T13
T_20 T_21 T22 T23;

Reply:
Single line will be ok?